src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -114,6 +114,18 @@ def mgr(self):
         return self._mgr_module
 
 
+class CephPoolConnection(object):
+    _connections = {}
```

Comment:
when are these connections closed and its instances pruned?

Reply:
the object returned by `rados.ioctx_open` doesn't need to be closed, I guess they all share the same connection that is closed here: https://github.com/openattic/ceph/pull/73/files#diff-0f85fb1dba9cfb52cf9e223346a6baecR104

But now that I think of, maybe caching the `ioctx` object does not provide a big performance benefit and this is just making things more complex.

Reply:
Ok, I looked into the implementation of the creation of the `ioctx` and I think it still makes sense to cache these objects.

Reply:
moving conversation to the current patch (this shows as outdated, for some reason)